mlir/test/Target/LLVMIR/vscale.mlir

```diff
@@ -0,0 +1,9 @@
+// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
+
+llvm.func @vscale_func()
+    vscale_range(2,8) {
+    // CHECK-LABEL: define void @vscale_func
+    // CHECK: attributes #{{.*}} = { vscale_range(2,8) }
+    llvm.return
+}
+
```

Comment:
nit: I think both test files have two newlines at the end. Can you drop one so that there is only one newline.